_sub/compute/eks-addons/dependencies.tf

```diff
@@ -14,7 +14,7 @@ locals {
     "1.21" = "v1.11.2-eksbuild.1"
     "1.22" = "v1.11.4-eksbuild.1"
     "1.23" = "v1.12.0-eksbuild.1"
-    "1.24" = "v1.12.1-eksbuild.2"
+    "1.24" = "v1.12.5-eksbuild.1"
```

Comment:
v1.12.2-eksbuild.1 is the recommended version for 1.24 and 1.24 when using mnaged add-ons. Reference: https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html